ROMFS/px4fmu_common/init.d/rc.mc_apps

```diff
@@ -4,18 +4,18 @@
 # att & pos estimator, att & pos control.
 #
 
-# The system is defaulting to INAV_ENABLED = 1
-# but users can alternatively try the EKF-based
-# filter by setting INAV_ENABLED = 0
-if param compare INAV_ENABLED 1
+# The system defaults to LPE if enabled, then
```

Comment:
I would prefer to only switch between LPE and EKF2 - there is no ongoing validation / testing work any more on INAV.